lib/wings/active_fedora_converter.rb

```diff
@@ -82,34 +79,34 @@ def active_fedora_class
     # then the id hasn't been minted and shouldn't yet be set.
     # @return [String]
     def id
-      id_attr = resource[:id]
-      return id_attr.to_s if id_attr.present? && id_attr.is_a?(::Valkyrie::ID) && !id_attr.blank?
+      return resource[:id].to_s if resource[:id]&.is_a?(::Valkyrie::ID) && !resource[:id].blank?
       return "" unless resource.respond_to?(:alternate_ids)
+
       resource.alternate_ids.first.to_s
     end
 
     def self.DefaultWork(resource_class)
       Class.new(DefaultWork) do
         self.valkyrie_class = resource_class
 
-        # nested attributes in AF don't inherit! this needs to be here until
-        # we can drop it completely.
+        resource_class.schema.each do |schema_key|
```

Comment:
A code comment for this block to state simple intention. This doesn't need the guard clause explanation, just tell me quickly what's happening.

Reply:
added. can you check if it looks okay?